cfgov/jinja2/v1/the-bureau/index.html

```diff
@@ -51,12 +53,12 @@
             },
             'video': {
                 'id': 'en0Iq8II4fA',
-                'url'   : 'https://www.youtube.com/watch?v=en0Iq8II4fA',
+                'url'   : youtubeURL,
                 'height': '320',
-                'width': '568'
+                'width' : '568'
```

Comment:
Minor, there should be no space before `:` but values themselves should left align to keep consistent w/ the JS formatting.